### DISCUSSION TITLE
Gitpod: Sometimes port is not public by default

### DISCUSSION THREAD

**Discussion by ngillux** [freeCodeCamp/classroom#331] [General]
During a paired programming session it was noticed that port 3001 (port for mock data) was not set to public  by default. This caused an issue when trying to get a response. If this issue arises, it's actually possible to just change the port to be public in Gitpod which resolved the issue during the session. In Gitpod in the same area the terminal is navigate to Ports > Identify the port which you need to be public under 'State' and make public if needed, you can do this by clicking on the lock icon (unlocked to make public, and locked to make private). This seemed to be a solution, but if there's anything else to add please share.

**Comment by GuillermoFloresV** [comment]
Hi @ngillux, thanks for documenting this. Based on our last talk, it seems that the solution I had does not work, I tried to replicate the success I had of just making the port public when notified of it running but I was not able to get it to work as intended. I'll continue to do research as to why that is, considering that it's worked for us in the past
